deploy/helm/values.yaml

```diff
@@ -1865,6 +1865,8 @@ controllers:
     enabled: true
   experimental:
     enabled: false
+  trace:
+    enabled: true
```

Comment:
How about setting it to false for now and re-enabling it when the performance issue is solved?

Reply:
Seems reasonable.